Cargo.toml

```diff
@@ -44,6 +44,7 @@ futures-core = "0.3.21"
 futures-util = "0.3.21"
 prost = "0.10.4"
 prost-derive = "0.10.1"
+serde = { version = "1.0.138", features = ["derive"] }
```

Comment:
There will be a new dependency [serde_derive](https://crates.io/crates/serde_derive), it's under [MIT ](https://choosealicense.com/licenses/mit)OR [Apache-2.0](https://choosealicense.com/licenses/apache-2.0) license, so it's OK.